### PR TITLE
fix: typo `scheme validate fail` should be `schema validate failed`

### LINKED ISSUE #914: bug: the schema verification prompt message contains typo
# Bug report

## Describe the bug

the schema verification prompt message contains typo

`scheme validate fail` should be `schema validate failed`

cc @nic-chen

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
fix  #914
typo:  `scheme validate fail` -> `schema validate failed`

### CODE REVIEW

File: api/internal/core/store/validate.go [hunk 1]
```diff
@@ -77,8 +77,8 @@ type APISIXJsonSchemaValidator struct {
 func NewAPISIXJsonSchemaValidator(jsonPath string) (Validator, error) {
 	schemaDef := conf.Schema.Get(jsonPath).String()
 	if schemaDef == "" {
-		log.Warnf("scheme validate failed: schema not found, path: %s", jsonPath)
-		return nil, fmt.Errorf("scheme validate failed: schema not found, path: %s", jsonPath)
+		log.Warnf("schema validate faileded: schema not found, path: %s", jsonPath)
+		return nil, fmt.Errorf("schema validate faileded: schema not found, path: %s", jsonPath)
```
Comment (idbeta) [inline api/internal/core/store/validate.go:81]: `faileded` -> `failed`

File: api/internal/core/store/validate.go [hunk 2]
```diff
@@ -123,25 +123,25 @@ func cHashKeySchemaCheck(upstream *entity.UpstreamDef) error {
 	if upstream.HashOn == "vars" {
 		schemaDef = conf.Schema.Get("main.upstream_hash_vars_schema").String()
 		if schemaDef == "" {
-			return fmt.Errorf("scheme validate failed: schema not found, patch: main.upstream_hash_vars_schema")
+			return fmt.Errorf("schema validate faileded: schema not found, patch: main.upstream_hash_vars_schema")
```
Comment (idbeta) [inline api/internal/core/store/validate.go:126]: dido

**Review by idbeta — COMMENTED** [review]
`faileded` -> `failed`

File: api/internal/core/store/validate.go (hunk 1 above)
Comment (nic-chen) [reply api/internal/core/store/validate.go]: fixed.

File: api/internal/core/store/validate.go (hunk 2 above)
Comment (nic-chen) [reply api/internal/core/store/validate.go]: fixed

**Review by moonming — APPROVED** [review]

**Review by membphis — APPROVED** [review]

**Pull request merged**